convert2rhel/actions/conversion/list_non_red_hat_pkgs_left.py

```diff
@@ -34,7 +34,7 @@ def run(self):
         super(ListNonRedHatPkgsLeft, self).run()
         loggerinst.task("List remaining non-Red Hat packages")
 
-        loggerinst.info("Listing packages not signed by Red Hat")
+        loggerinst.info("Listing packages not signed by Red Hat.")
```

Comment:
This will be unique as most don't have a dot in them, but definitely more valid with a dot than without. Just an FYI

Reply:
Most info messages had a full stop. For consistency I've added the full stop to all of them (and to other levels such as debug or critical as well).